### PR TITLE
svn mailbox command

### PR DESCRIPTION
**Pull request by sphinxc0re**

### CODE REVIEW

File: runtime/src/mailbox_api.rs [hunk 1]
```diff
@@ -264,3 +268,11 @@ pub struct FwInfoResp {
...
     // status.
 }
+
+/// SVN response
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct GetSvnResp {
+    pub runtime_svn: u32,
+    pub fmc_manifest_svn: u32,
+}
```
Comment (jhand2) [inline runtime/src/mailbox_api.rs:278]: Instead of making a new command, I think we should add these to the FW_INFO command. Thanks!

Comment (sphinxc0re) [reply runtime/src/mailbox_api.rs]: Done!

File: error/src/lib.rs [hunk 2]
```diff
@@ -20,410 +20,365 @@ use core::num::NonZeroU32;
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub struct CaliptraError(pub NonZeroU32);
 impl CaliptraError {
-    const fn new_const(val: u32) -> Self {
```
Comment (jhand2) [inline error/src/lib.rs:23]: Are the changes in this file required? I would prefer to avoid including this change in this PR if we don't have to.

Comment (sphinxc0re) [reply error/src/lib.rs:23]: Sure, I will put it in a separate PR then.

**Pull request merged**